cost-analyzer/values.yaml

```diff
@@ -401,10 +401,24 @@ kubecostFrontend:
   #   large_client_header_buffers 4 64k;
   # hideDiagnostics: false  # useful if the primary is not monitored. Supported in limited environments.
   # hideOrphanedResources: false  # OrphanedResources works on the primary-cluster's cloud-provider only.
+
+  # set to true to set all upstreams to use <service>.<namespace>.svc.cluster.local instead of just <service>.<namespace>
+  use_default_fqdn: false
```

Comment:
```suggestion
  useDefaultFQDN: false
```

Reply:
...and related references in the template.

Reply:
Woops. Missed this one. Sorry bout that... Updated all refs